web/src/info_overlay.ts

```diff
@@ -359,3 +360,11 @@ export function show(target: string | undefined): void {
         toggler!.goto(target);
     }
 }
+
+function process_message_formatting_layout($rendered_html: JQuery): void {
+    $rendered_html.find("p").addClass("preserve-message-formatting-layout");
```

Comment:
I think you can pass a selector like "p, ul, ol, pre, blockquote" to `.find()`, though if these are all immediate children, `.children()` would be the better jQuery function, I think.